apps/manager_app/onboarding/onboarding.c

```diff
@@ -94,7 +113,7 @@ static onboarding_ctx_t onboarding_ctx = {.static_screen = false,
 
 static const flow_step_t onboarding_flow = {
     .step_init_cb = onboarding_initialize,
-    .p0_cb = NULL,
+    .p0_cb = ignore_p0_handler,
```

Comment:
can directly assign `ignore_p0_event` instead using the wrapper `ignore_p0_handler`

Reply:
The wrappers were added to avoid `events.h` include in `core_error.h` file. As the `ignore_p0_event` would need to have a prototype same as `ignore_p0_handler`